spec/services/coronavirus_pages/model_builder_spec.rb

```diff
@@ -46,6 +46,26 @@
         model_builder.page
         expect(sub_section.content).to include(section["sub_sections"].first["list"].first["label"])
       end
+
+      it "creates sub_sections with a position that reflects their order in the content item" do
+        input_order =
+          source_sections.pluck("title").each_with_index.to_h.invert
+
+        # {0=>"How to protect yourself and others",
+        #  1=>"Employment and financial support",
+        #  2=>"School closures, education, and childcare",
+        #  3=>"Businesses and other organisations",
+        #  4=>"Healthcare workers and carers",
+        #  5=>"Travel",
+        #  6=>"How coronavirus is affecting public services",
+        #  7=>"How you can help",
+        #  8=>"Coronavirus (COVID-19) cases in the UK"}
```

Comment:
I'm not sure these comments are useful (the actual content of the comments isn't important, and if someone wants to look it up, it would be better to look at the source yaml document), and they will be difficult to maintain. I'd suggest removing this comment.